lib/bolt/bolt_option_parser.rb

```diff
@@ -6,6 +6,16 @@
 
 module Bolt
   class BoltOptionParser < OptionParser
+    EXCLUDES = { 'apply' => %w[params],
```

Comment:
Not sure if this is the best way to exclude options from being shown, but it feels a bit cleaner than separate methods that toggle the hide variable. This approach lets you add an option to a command's array, and it will then be removed from the help text.